src/python/py_utils.cpp

```diff
@@ -283,4 +301,65 @@ ov::genai::OptionalGenerationConfig update_config_from_kwargs(const ov::genai::O
     return res_config;
 }
 
+    
+bool generation_config_param_to_property(std::string key, py::object value, ov::AnyMap& map) {
```

Comment:
maybe we can reuse this function in `update_config_from_kwargs` ? E.g. convert params to AnyMap, then call GenerationConfig::update_generation_config

Reply:
I'm ok to merge as is for releases/2024/5 and improve on master branch

Reply:
> can we convert kwargs to AnyMap and then call GenerationConfig::update_from_map ?

Yes, did it inside `update_config_from_kwargs()` method.